Pipfile

```diff
@@ -4,17 +4,23 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-coveralls = "*"
+codecov = "*"
+black = "*"
 unittest-xml-reporting = "*"
 flake8 = "*"
```

Comment:
if we're using black, do we still need this?

Reply:
Yep, there are still a couple of things that black lets through that we wouldn't want. Import order is one of them (and fixed using isort)